### PR TITLE
Pract.04: Add 01-03 | Daria Lazarova

### PR DESCRIPTION
**Pull request by DariaLaz**
Daria Lazarova
2 8MI0600140

### CODE REVIEW

File: SI/Sem.04/Pract.04/Task01.cpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿#include <iostream>
+
+using namespace std;
+
+int sum(int a, int b) {
+    return (a + b);
```
Comment (Justsvetoslavov) [inline SI/Sem.04/Pract.04/Task01.cpp:6]: Brackets are not needed.

File: SI/Sem.04/Pract.04/Task02.cpp [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿#include <iostream>
+
+using namespace std;
+
+bool isEven(int num) {
+    return (num % 2 == 0);
```
Comment (Justsvetoslavov) [inline SI/Sem.04/Pract.04/Task02.cpp:6]: Brackets are not needed.

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿#include <iostream>
+
+using namespace std;
+
+bool isPositive(int num) {
+    return (num > 0);
```
Comment (Justsvetoslavov) [inline SI/Sem.04/Pract.04/Task03.cpp:6]: Brackets are not needed.

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+}
+bool isNegative(int num) {
+    return (num < 0);
```
Comment (Justsvetoslavov) [inline SI/Sem.04/Pract.04/Task03.cpp:9]: Brackets are not needed.

**Review by Justsvetoslavov — APPROVED** [review]

**Review by GeorgiTerziev02 — APPROVED** [review]

**Review by Justsvetoslavov — APPROVED** [review]
Added a little bit of refactoring, in order to merge.

**Pull request merged**